connectors/aws/aws-comprehend/src/main/java/io/camunda/connector/comprehend/model/ComprehendSyncRequestData.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ *       under one or more contributor license agreements. Licensed under a proprietary license.
+ *       See the License.txt file for more information. You may not use this file
+ *       except in compliance with the proprietary license.
+ */
+package io.camunda.connector.comprehend.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import io.camunda.connector.generator.java.annotation.TemplateSubType;
+import jakarta.validation.constraints.NotNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@TemplateSubType(id = "sync", label = "Sync")
+public record ComprehendSyncRequestData(
+    @TemplateProperty(
+            group = "input",
+            label = "Text",
+            description = "The document text to be analyzed.")
+        @NotNull
+        String text,
+    @TemplateProperty(
+            id = "sync.documentReadAction",
+            label = "Document read action",
+            group = "input",
+            type = TemplateProperty.PropertyType.Dropdown,
+            defaultValue = "TEXTRACT_DETECT_DOCUMENT_TEXT",
```

Comment:
Is it "TEXTRACT_DETECT_DOCUMENT_TEXT" ? Or rather comprehend related?

Reply:
part of comprehend https://docs.aws.amazon.com/comprehend/latest/APIReference/API_DocumentReaderConfig.html#comprehend-Type-DocumentReaderConfig-DocumentReadAction